### DISCUSSION TITLE
How to prevent download the default model files?

### DISCUSSION THREAD

**Discussion by 925-Studio** [lllyasviel/Fooocus#1002] [Q&A] [answered]
I actually have those files in the folder, I just renamed them to keep them organized. But after I renamed those files, Fooocus tends to download them again on startup. It's very annoying and is there something I can do? Thanks in advance.

**Comment by lllyasviel** [comment, marked as answer]
see also "default_model" and "checkpoint_downloads" in config_modification_tutorial.txt and related desc in readme

**Comment by kidooo26** [comment]
add on run.bat --disable-preset-download
